.ci/pytorch/win-test-helpers/installation-helpers/install_oneapi_bundle.bat

```diff
@@ -0,0 +1,37 @@
+set ONEAPI_PARENT_DIR=C:\Program Files (x86)\Intel
+set INTEL_ONEAPI_PYTORCH_BUNDLE_URL=https://registrationcenter-download.intel.com/akdlm/IRC_NAS/5ca2021d-dd1a-4ab1-bd52-758fe63cf827/w_intel-for-pytorch-gpu-dev_p_0.5.2.19_offline.exe
```

Comment:
What's the timeline to uplift intel-for-pytorch-gpu-dev to 0.5.3?

Reply:
Whenever 0.5.3 publicly available, I will update it. Thank you.